packages/next-plugin/src/index.ts

```diff
@@ -115,9 +115,7 @@ export const createVanillaExtractPlugin = (
         dir,
         resolvedNextConfig.experimental?.appDir ?? false,
```

Comment:
Should this also be removed?
```suggestion
```

Reply:
I think this can be kept so that it is compatible with Next.js 13.4.x and lower versions.
References: https://github.com/vercel/next.js/blob/v13.4.19/packages/next/src/lib/find-pages-dir.ts#L50